common/tags/pom.xml

```diff
@@ -34,6 +34,14 @@
     <sbt.project.name>tags</sbt.project.name>
   </properties>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.scalatest</groupId>
+      <artifactId>scalatest_${scala.binary.version}</artifactId>
+      <scope>compile</scope>
+    </dependency>
+  </dependencies>
```

Comment:
I hit the same issue after fetching the latest master. I did clean and rebuild. It still has the issue.